cognite/client/_api/data_modeling/instances.py

```diff
@@ -631,6 +631,7 @@ def apply(
             resource_cls=_NodeOrEdgeApplyResultAdapter,  # type: ignore[type-var]
             extra_body_fields=other_parameters,
             input_resource_cls=_NodeOrEdgeApplyAdapter,  # type: ignore[arg-type]
+            limit=100,
```

Comment:
the limit is set in the API client base class. I assume you want to split up request to have a max 100, and not limit all method calls to have 100 limit.